fastapi_pagination/cursor.py

```diff
@@ -75,6 +75,11 @@ def to_raw_params(self) -> CursorRawParams:
 class CursorPage(AbstractPage[T], Generic[T]):
     items: Sequence[T]
 
+    current_page: str = Field(..., description="Cursor to refetch the current page")
+    current_page_backwards: str = Field(
+        ...,
+        description="Cursor to refetch the current page starting from the last item",
+    )
```

Comment:
Let's make those fields optional cause they can't be unavailable for other extenstions

Reply:
Thanks for the feedback! Is this a future proofing thing? I didn't see anything but the sqlalchemy extension using this.

Reply:
There are other extensions that use `CursorPage`:
https://github.com/uriyyo/fastapi-pagination/blob/main/fastapi_pagination/ext/cassandra.py